### ISSUE TITLE
libicui18n.so.54

### ISSUE THREAD

**Issue by Collinthegeek** [papyros/papyros-shell#62]
I installed Papyros-shell from the repository and now I'm getting 

$ papyros-shell
papyros-shell: error while loading shared libraries: libicui18n.so.54: cannot open shared object file: No such file or directory

**Comment by 29jm** [comment]
This post seems relevant, assuming you're on Arch https://bbs.archlinux.org/viewtopic.php?id=160271

**Comment by 29jm** [comment]
The package containing this library is `icu` on Arch, try installing that

**Comment by danikron** [comment]
The `icu` package has been updated to `icu 55.1-1` which brings `libicui18n.so.54` to `libicui18n.so.55` and `libicuuc.so.54` to `libicuuc.so.55`.

**Comment by Cheapskate01** [comment]
Is there a way to download a previous version of icu?

**Comment by danikron** [comment]
Probably, but I don't think that's a very good idea. Likely there are many other packages that make calls to `icu` and which would also have to be downgraded. It's probably best to wait until `qt5-base-git` is updated to make the correct calls, or `papyros-shell` is altered to use the `qt5-base` package instead.

**Comment by Collinthegeek** [comment]
Or make symlinks to fake it

**Comment by Collinthegeek** [comment]
@cheapskate01@danikron /\

**Comment by iBeliever** [comment]
Is this still an issue after rebuilding `qt5-base-git` after the ice package was updated? If so, this should get reported upstream so the Qt guys can fix it.

**Comment by iBeliever** [comment]
I rebuilt Qt 5 from the AUR and it everything works fine now (testing with the Green Island branch). So to fix this problem, either rebuilt from the AUR, or install the packages I built. Our new Papyros build server & repo is at http://45.55.156.48 (repo path = http://45.55.156.48/repos/$repo/$arch). Once I finish building all the packages, I'll move that to the public domain http://build.papyros.io and remove the existing http://dash.papyros.io server.

@Collinthegeek because you were the one to open this issue, could you please confirm that my solution above works correctly for you and then close the issue?

**Comment by Cheapskate01** [comment]
Now it says something about libproxy.so.1.

**Comment by iBeliever** [comment]
@Cheapskate01 I'm guessing you used the papyros-shell package from the old repo?

**Comment by Cheapskate01** [comment]
I honestly don''t know. I changed the pacman.conf file and ran the sudo pacman -Syu command. It updated qt5-base-git. How would I update the papyros shell.
I'm used to debian, not arch so I seem like a newb.

**Comment by Cheapskate01** [comment]
wait, let me run a quick update.

**Comment by Cheapskate01** [comment]
How does one update this? I tried and it doesn't find target

**Comment by danikron** [comment]
Using Pacman you need to make sure that the entry for the new repo is above the AUR repo in pacman.conf before reinstalling. Alternatively you could use Yaourt, which should let you pick which repo to install from.

**Comment by bigfoot90** [comment]
@Collinthegeek I've solved with ```pacman -S libproxy```

**Comment by kyuz0** [comment]
Hi. So, I just installed a clean copy of Arch today. I originally tried to use the repository on the website, but I got the libicui18n.so.54 error. I then removed that repository and installed the new one you pointed out to. That seemed to trigger some qt updates. However, the papyros-shell was not loading, different error related to missing qt platform plugin xcb. Then, I removed papyros-shell and tried to reinstall from new repo... but it says target does not exist... not sure how to proceed. 

I really wanted to try this out, not sure if there's a quick fix or something obvious I'm missing here. This is an awesome project, looking forward to the alpha :))
